examples/component/map.js

```diff
@@ -1,58 +1,49 @@
 Ext.require([
-    'GeoExt.component.Map',
-    'Ext.panel.Panel',
-    'Ext.Viewport'
+    "GeoExt.component.Map",
+    "Ext.panel.Panel",
+    "Ext.Viewport"
 ]);
 
-var olMap,
-    mapComponent,
-    mapPanel;
```

Comment:
@chrismayer, thanks for the reminder. I didn't think of that. But that kind of proves a more explicit statement to export to the global object may be more clear.